packages/nitro-protocol/ganache/index.ts

```diff
@@ -0,0 +1,36 @@
+import {configureEnvVariables, GanacheServer} from '@statechannels/devtools';
+import Koa = require("koa");
+import Router = require('koa-router');
+import {deploy} from '../deployment/deploy-test';
+import {logger} from './logger';
+// configureEnvVariables();
```

Comment:
```suggestion
```

Reply:
Should this be run? or should it be deleted?

Reply:
This is a temporary commenting out. I haven't thought much about how this configuration should be handled as of yet (getting to that soon so will update this line).